governance/tag-governance.md

```diff
@@ -0,0 +1,541 @@
+- SHOULD be an escalation point for technical discussions and decisions in the subproject.  
+- SHOULD set milestone priorities.   
+- SHOULD ensure a healthy process for discussion and decision making is in place. MAY make decisions
+  to resolve conflicts.
+
+##### Activity Expectations
+
+- Subproject Leads serve 1 year terms, and  MAY submit for re-election.  
+- Subproject Leads MAY decide to step down at any time and propose a replacement to complete their
+  term.   
+  - The candidate SHOULD be supported by a majority of the current active leads, with support backed
+    up by documentation.  
+  - The candidate is REQUIRED to be approved by the parent TAG’s Tech Leads.  
+- Subproject Leads taking an extended leave of 1 or more months SHOULD coordinate with other leads
+  to ensure the role is adequately covered during their leave.  
+- Subproject Leads going on leave for 1-3 months MAY work with other Leads to identify a temporary
+  replacement.  
+  - Active Leads are REQUIRED to notify their parent governing body (e.g., TAG notifying their TOC
+    liaison) and receive confirmation of notice before pursuing removal of an inactive or Lead found
+    to be in dereliction of duty.  
+  - Inactive Leads MAY be removed through a super-majority vote of the parent governing body if
+    there is no other explicitly defined removal policy.  
+  - If there is no defined removal policy and not enough active parent governing body leads to hold
+    a vote, the active members SHOULD escalate to the next-level parent governing body, where those
+    members may count as additional voters.
+
+##### Escalations
+
+- Lead membership disagreements MAY be escalated to their parent governing body, the CoCC as
+  applicable, or in the event of further dispute, up to the top governing body itself (TOC).
+
+#### Other Roles
+
+Subprojects MAY have other roles, but are required to be documented in the subproject charter and
+adhere to the requirements outlined in the Other Roles section of the CNCF Community Group
+Leadership Requirements.
+
+**NOTE:** Roles that assume some of the responsibilities of the Subproject Lead role MUST be
+described in the Subproject charter and REQUIRE TAG Lead approval.
+
+Roles that do not assume any Subproject Lead responsibilities may be created at the Subproject Leads
+discretion as long as they adhere to the following requirements:
+
+- MUST be documented with a description, scope, and responsibilities.  
+- MUST have a documented lifecycle policy including how they are appointed, any potential terms, and
+  an inactivity policy.  
+- MUST be tracked alongside the other TAG metadata.  
+- MUST adhere to the [Technical Leadership Principles].  
+- SHOULD remain active and responsive in their Roles.
+
+### Formation
+
+Subprojects require a charter describing their purpose, scope, roles, and any regular deliverables
+or services they provide.  This charter MUST be approved by the TOC. 
```

Comment:
Listing requirements outside of the requirement section may result in them being missed later on.